add information about how we work (#1357)
18F/tts-tech-portfolio: how_we_work/README.md

# How we work

[The team's Handbook page](https://handbook.tts.gsa.gov/tech-portfolio/) contains general information about the team for external consumption, while this folder has information for the team members ourselves. Specific documents of note:

- [Roles](roles.md): positions held within the Tech Portfolio
- [Workflow](workflow.md): how we keep our work organized

## Work time

Our working schedule is flexible:

- We don't have team-wide core hours.
- Aside from meetings, your schedule is yours.
- You are expected to keep your calendar up-to-date.
- We expect response to messages within a few working hours.
- You are expected to work forty hours per week, no more, no less.
  - If you have too much or too little to do, tell the Director.

See [meetings](policy/meetings.md) for more details.

> ## Team maxims

Author: Are these useful to include? Was trying to think of things that distinguish us from other teams.

- Open is better than closed.
  - For information that isn't [sensitive](https://handbook.tts.gsa.gov/sensitive-information/), make it as public as possible/practical.
- [There's no such thing as user error.](https://medium.com/swlh/theres-no-such-thing-as-user-error-76b564454234)
- If you see something, fix something.

> - We have way more work to do than people to do it, so we are constantly on the lookout for ways to multiply our impact and things to take off our plates.

Reviewer: I think Dave Z would disagree with this and instead we should have the people to cover the things we need to do.

That said, it's correct at the moment and is a useful framework to keep in mind during prioritization. Maybe I would rephrase 🤷 

```suggestion
- We have way more requests than we have people to respond, so we are constantly on the lookout for ways to multiply our impact and things to take off our plates.
```

> - We have more asked of our team than we are able to fulfill, so we are constantly on the lookout for ways to multiply our impact and things to take off our plates.

Author: @adborden Better?

Reviewer: lgtm, thanks!
